tools/not.cpp

```diff
@@ -49,13 +55,22 @@ int main(int argc, const char **argv) {
   if (argc == 0)
     return 1;
 
+  int result;
+#if !defined(TARGET_OS_IPHONE)
   std::stringstream ss;
   ss << argv[0];
   for (int i = 1; i < argc; ++i)
     ss << " " << argv[i];
   std::string cmd = ss.str();
-
-  int result = std::system(cmd.c_str());
+  result = std::system(cmd.c_str());
+#else
+  char* const* environ = NULL;
+  pid_t pid;
+  if (posix_spawn(&pid, argv[0], NULL, NULL, argv, environ))
```

Comment:
could you put a NULL instead of an environ?

Reply:
Done